src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -181,52 +181,96 @@ $(document).ready(function () {
                 newOption: true
             }
         }
-    }).on('select2:select', function (e) {
-        if (e.params.data.newOption) {
-            var listname = searchLookup.data('listname');
-            document[listname]._startRecord.value = 0;
-            document[listname].submit();
-        } else {
-            var id = e.params.data.id.trim();
-
-            if ('' === id) {
-                return;
-            }
-
-            switchRecord(id);
-        }
     }).on('select2:unselect', function (e) {
         // :unselect and submit() doesn't transmit the empty searchlookup-Field. Therefore, the value of
         // _search_word is not overwritten (not reset) in the session. With an additional hidden field it works.
         const form = createSearchWordInputWithValue();
         form.submit();
-    }).on('select2:closing', function(e, what) {
-        // If the select2 widget's autocomplete list is being closed without any selection made
-        // we will check whether a search term has been typed into the search field.
-        // If the search field contains a value an additional input field will be added to the form. After closing the
-        // autocomplete list by hitting the enter key the `select` listener will be triggered which will handle the
-        // input as `newOption`.
-
-        // Known drawback: The search field associated to the current select2 widget can not be directly determined as
-        // select2 does not expose any way to retrieve it. Therefore the DOM will be queried for the
-        // search field's selector.
-        // The query will return the first search field found in the DOM which might not be the desired one.
-        // Yet as select2 creates a new search field on activating the widget and removes it when unnecessary there
-        // should be only one search field present in the DOM.
+    }).on('select2:closing', function(e) {
+         /* There is no handler attached to the `select` event. Instead the `closing` handler will also manage the
```

Comment:
Better describe what should be achieved here (and how). Like:

Only monitoring closing here because this simplifies X

Reply:
Gefällt mir. Done.